hash/nes.xml

```diff
@@ -24626,6 +24626,43 @@ license:CC0-1.0
 		</part>
 	</software>
 
+	<software name="micromages">
+	  <description>Micro Mages</description>
+	  <year>2019</year>
+	  <publisher>Morphcat Games</publisher>
+	  <info name="serial" value="NES-MM-MCG" />
+	  <info name="release" value="20190212" />
+	  <part name="cart" interface="nes_cart">
+		<feature name="slot" value="nrom" />
+		<feature name="pcb" value="NES-NROM-256" />
+		<feature name="mirroring" value="horizontal" />
+		<dataarea name="prg" size="32768">
+		  <rom name="Micro Mages prg" size="32768" crc="1f4aa740" sha1="c1f0e973815687e06086e57a615abc116aa37e61" />
+		</dataarea>
+		<dataarea name="chr" size="8192">
+		  <rom name="Micro Mages chr" size="8192" crc="24144d94" sha1="e08bec8c93dce07262be37c7cceb7019817bea69" />
+		</dataarea>
+	  </part>
+	</software>
+
+	<software name="micromages2">
+	  <description>Micro Mages Second Quest</description>
+	  <year>2022</year>
+	  <publisher>Morphcat Games</publisher>
+	  <info name="release" value="20220915" />
```

Comment:
This is less than three years old, and only available as a paid download.  As such, it should not be added to the software list.  We generally try to be nice to software developers.

Reply:
Makes sense, feel free to reject the merge. The two games come as a bundle.